contracts/core/EntryPointSimulations.sol

```diff
@@ -129,8 +119,8 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
         return ExecutionResult(
             opInfo.preOpGas,
             paid,
-            data.validAfter,
-            data.validUntil,
+            validationData,
```

Comment:
Is it okay that the returned structure does not explicitly say if the call will be accepted or not, and requires some client logic to figure that out?

Reply:
the method call is for the bundler only. 
(in the past, we did partial work to parse it in solidity, but it was partial anyway and required the bundler to check it - then why bother to do part of the parsing in solidity, and the rest in the bundler's native code? it can be all native.